validate.sh

```diff
@@ -1,7 +1,15 @@
 #!/usr/bin/env bash
-node_modules/.bin/stac-node-validator \
-    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/template/schema.json=extensions/template/schema.json \
-    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/camera/schema.json=extensions/camera/schema.json \
-    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/quality/schema.json=extensions/quality/schema.json \
-    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/linz/schema.json=extensions/linz/schema.json \
-    ${@+"$@"}
+
+validator_command=(
+    node_modules/.bin/stac-node-validator
+    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/template/schema.json=extensions/template/schema.json
+    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/camera/schema.json=extensions/camera/schema.json
+    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/quality/schema.json=extensions/quality/schema.json
+    --schemaMap=https://linz.github.io/stac/_STAC_VERSION_/linz/schema.json=extensions/linz/schema.json
+)
+"${validator_command[@]}" extensions/*/examples/*.json
+
+for path in extensions/*/non-examples/*.json
+do
+    ! "${validator_command[@]}" "$path"
+done
```

Comment:
Would you be OK to configure your IDE to add a newline at EOL? That simplifies the diffs when dealing with merges and the like. The setting is called "Ensure every saved file ends with a line break" in IDEA.